packages/annotations/src/store/selectors.js

```diff
@@ -61,5 +65,7 @@ export const __experimentalGetAnnotationsForRichText = createSelector(
  * @return {Array} All annotations currently applied.
  */
 export function __experimentalGetAnnotations( state ) {
-	return state.all;
+	return flatMap( state, ( annotations ) => {
+		return annotations;
+	} );
```

Comment:
This is equivalent to `flatMap( state, identity )`, which can be reduced to `flatten( state )`, unless I misread.